Makefile

```diff
@@ -229,6 +229,10 @@ docker-push-dns-server:
 docker-push-chaos-kernel:
 	docker push "${IMAGE_REGISTRY_PREFIX}pingcap/chaos-kernel:${IMAGE_TAG}"
 
+bin/chaos-multiversion-helper: SHELL:=$(RUN_IN_DEV_SHELL)
+bin/chaos-multiversion-helper: images/dev-env/.dockerbuilt
```

Comment:
If we use this command, the binary can't be used in mac directly.

Reply:
Oops. But maybe I shouldn't compile it inside the dev env 🤔

Reply:
I think run `chaos-multiversion-helper` inside the dev-dev is a better solution. We could add an alias for `docker run xxx chaos-multiversion-helper` command, and users could still use `chaos-multiversion-helper` command in the document to execute.

Reply:
I add a single command: 

```
OLD_VERSION="v1alpha1" NEW_VERSION="v1alpha2" make migrate-version
```

To run inside a container. It should fix this problem :smile:

Reply:
I agree with this solution.